praetorian_cli/handlers/workflow.py

```
@@ -0,0 +1,330 @@
+
+    # Build example command
+    example_args = []
+    for arg in meta.required_args:
+        if arg == "cve_id":
+            example_args.append("cve_id=CVE-2024-1234")
+        elif arg == "template":
+            example_args.append("template=./template.yaml")
+        else:
+            example_args.append(f"{arg}=<value>")
+
+    click.echo(f"\nUsage:")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)